app/views/course/assessment/submission/submissions/_submission.html.slim

```diff
@@ -1,11 +1,11 @@
 = content_tag_for(:tr, submission) do
-  td.col-md-4.col-xs-4 = link_to_course_user(submission.course_user)
+  td.col-md-4.col-xs-4 = link_to_course_user(course_user)
   td.col-md-3.col-xs-3
     = link_to edit_course_assessment_submission_path(current_course, submission.assessment,
                                                      submission) do
       = Course::Assessment::Submission.human_attribute_name(submission.workflow_state)
   td
-    = submission.grade.to_s + ' / ' + submission.assessment.maximum_grade.to_s
+    = submission.grade.to_s + ' / ' + @assessment.maximum_grade.to_s
```

Comment:
Maybe we document why we are using it here so that devs will not accidentally copy and assume that we should use instance variables in partials?